### PR TITLE
Adjusting best practice checkout guide layout widths

### PR DESCRIPTION
**Pull request by cdwinsnes**

### CODE REVIEW

**Comment by WebDevBring** [comment]
[trello.com/c/FYyqAeKP](https://trello.com/c/FYyqAeKP)

File: css/bestpractice.css [hunk 1]
```diff
@@ -6,12 +6,10 @@
     max-width: max-content;
 
     & > div {
-        flex: 1 2 24rem;
-        max-width: max-content;
+        max-width: 40rem;
```
Comment (halvorsanden) [inline css/bestpractice.css:9]: Any reason not to flex these anymore?

Comment (cdwinsnes) [reply css/bestpractice.css:9]: It's work in progress 😅

Comment (halvorsanden) [reply css/bestpractice.css:9]: Ah, sorry for jumping the gun 😳

**Review by halvorsanden — APPROVED** [review]

**Pull request merged**